src/config/postgres.ts

```diff
@@ -0,0 +1,11 @@
+import { Pool } from 'pg';
+
+const pool = new Pool({
+  host: process.env.PG_HOST || 'localhost',
+  port: parseInt(process.env.PG_PORT || '5433'),
```

Comment:
Is the default RDS port 5433 or 5432? Can we have it match the default?

Reply:
yes the RDS default is 5432, so let's change it